src/service/auth-service.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+const loginUser = async (loginCreds) => {
+  const loginUrl = '/api/auth/login';
+  try {
+    const { data, status } = await axios.post(loginUrl, loginCreds);
+    if (status === 200)
+      return data;
+    if (status === 500)
```

Comment:
dont do 500 and 400 case, you can just let them pass down to catch

Reply:
I understood this and removed in my next pr.